language/stdlib/modules/genesis.move

```diff
@@ -74,7 +72,7 @@ module Genesis {
         LibraAccount::create_treasury_compliance_account<LBR::T>(
             association,
             tc_addr,
-            tc_auth_key_prefix,
+            copy dummy_auth_key_prefix,
```

Comment:
Not sure I quite follow this change -- I can't see where the key is rotated to the correct key. Now that rotate key takes a `signer` maybe do those all at the bottom of this?

Reply:
Ah, I probably should have made this change somewhere else. The old code was wrong (but harmlessly). The problem is that (1) all the accounts we create in genesis get rotated to the genesis auth key and (2) even if we wanted a different auth key for the TC account, passing `tc_auth_key_prefix` into the account creation wouldn't work. The problem is  that even if we get to choose `tc_auth_key_prefix`, we won't know the private key that can sign for `auth_key_prefix | 0xb1e55ed`....